_data/icons/chain_verse.json

```diff
@@ -0,0 +1,8 @@
+[
+  {
+    "url": "ipfs://QmQyJt28h4wN3QHPXUQJQYQqGiFUD77han3zibZPzHbitk", 
+    "width": 1000,
+    "height": 1628,
```

Comment:
identify QmQyJt28h4wN3QHPXUQJQYQqGiFUD77han3zibZPzHbitk         18:19:23
QmQyJt28h4wN3QHPXUQJQYQqGiFUD77han3zibZPzHbitk PNG 1000x1436 1000x1436+0+0 8-bit sRGB 90980B 0.000u 0:00.000

Reply:
set the correct width and height.
thank you